docs/workflow.rst

```diff
@@ -36,7 +36,8 @@ Datamodel.py
 
 Before running the BEAST, you will need to modify this file to specify
 the required parameters for generating models and fitting data.
-These parameters are described in beast setup documenation (link needed).
+These parameters are described in the beast `setup documentation <beast_setup.rst>`_.
```

Comment:
Have you checked that this provides the correct link when running 'python setup.py build_docs' in the main beast directory?  I have not had great success getting internal links to work.  So, I thought I'd ask before merging.  :-)